bref

```diff
@@ -359,7 +367,7 @@ $app->command('dashboard [--host=] [--port=] [--profile=] [--stage=]', function
     return $process->getExitCode();
 })->descriptions('Start the dashboard');
 
-$app->command('bref.dev [--profile=] [--stage=]', function (string $profile = 'default', string $stage = null, SymfonyStyle $io) {
+$app->command('bref.dev [--profile=] [--stage=]', function (?string $profile, string $stage = null, SymfonyStyle $io) {
```

Comment:
I am afraid that this change will make the `$profile` variable empty, which may mess up the `serverless info` command built below (the `--aws-profile` is always passed to the command, and now it will not have a value).

Does this still work?

Reply:
Good catch. I've added the usual credential chain (`--profile` then `AWS_PROFILE` then `default`). Otherwise the env var is ignored.

Reply:
Awesome thanks!